engine/flink/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/source/flink/FlinkKafkaSource.scala

```diff
@@ -82,6 +87,13 @@ class FlinkKafkaSource[T](preparedTopics: List[PreparedKafkaTopic],
     deserializationSchema.deserialize(record)
   }
 
+    override def testParametersDefinition: List[Parameter] = testParametersInfo.uiParameters
```

Comment:
What happened with indention here?

Reply:
No idea, it seems it needed some space :drum: :smile: - Fixed